docs/pages/eas-update/asset-selection.mdx

```diff
@@ -28,15 +28,40 @@ To use this feature, include the new property `extra.updates.assetPatternsToBeBu
 
 In this case, all **.png** files in all subdirectories of **app/images** will be included in updates.
 
-Assets that do not match any of the file patterns will still resolve in the Metro bundler, but will not be uploaded to the updates server. If making use of this feature, a developer needs to be certain that assets not included in updates are built into the native build of the app.
-
 If this new property is not in your app config, all assets resolved by the bundler will be included in updates (as per SDK 49 and earlier behavior).
 
+### Ensuring that an update contains all assets needed by your application
+
+When using this feature, assets that do not match any of the file patterns will still resolve in the Metro bundler, but will not be uploaded to the updates server. A developer needs to be certain that assets not included in updates are built into the native build of the app.
+
+If you are building your app locally, or have access to the correct build that you are publishing updates for (with the same **runtimeVersion**), then the **expo-updates** CLI contains a command, **assets:verify**, that will allow you to check whether all required assets will be included when you publish an update:
```

Comment:
```suggestion
If you are building your app locally or have access to the correct build for publishing updates (with the same **runtimeVersion**), then you can use the `npx expo updates assets:verify` command. It allows you to check whether all required assets will be included when you publish an update:
```

- Instead of describing about `expo-updates` CLI, let's say the complete command (not sure how much we document npx expo-updates as a CLI before this particular doc)
- Let's also break the last sentence for better readaiblity.